### DISCUSSION TITLE
Rename Device Blocker to Endpoint Blocker

### DISCUSSION THREAD

**Discussion by elueckel** [elueckel/Unifi-Toolbox#51] [Ideas]
Ich denke ich will den Device Blocker noch in Endpoint Blocker umbenennen (also nur Name). 

Endpoints sind Geräte während Device Teile von Unifi sind. Dann es er auch gleich mit dem Endpoint Monitor. Verzeichnis usw. würde ich gleiche lassen, sonst gibt es wieder Schmerzen beim Upgrade?

Was denkst du @Brovning

**Comment by Brovning** [comment]
Ah, jetzt verstehe ich das auch mal. :-)
Ja, können wir machen.
Soll ich oder machst du?

**Comment by elueckel** [comment]
Mache ich :-)

**Reply by Brovning** [reply]
Hier recht kannst du die Diskussion direkt in ein Ticket umwandeln, sofern du das noch nicht gesehen hast:
https://github.com/elueckel/Unifi-Toolbox/issues/new?created_from_discussion_number=51

**Comment by Brovning** [comment]
topic closed with #52
